### PR TITLE
#382: TopoJson.get_bounds() returns [lon,lat] instead of [lat,lon]

### LINKED ISSUE #382: TopoJson.get_bounds() returns [lon,lat] instead of [lat,lon]
`TopoJson._get_self_bounds()` returns `[[xmin, ymin], [xmax, ymax]]` i.e. a bounding box defined by longitude,latitude points, rather than lat,lon points.

This is contrary to `GeoJson._get_self_bounds()` as well as contrary to the expectations of `FitBounds`:
> bounds: list of (latitude, longitude) points

### PR DESCRIPTION
**Pull request by eddies**
Fixes #382

### CODE REVIEW

File: tests/test_folium.py [hunk 1]
```diff
@@ -470,8 +470,8 @@ def test_topo_json(self):
         assert ''.join(topojson_str.split())[:-1] in ''.join(out.split())
 
         bounds = self.map.get_bounds()
-        assert bounds == [[-124.56617536999985, 41.99187135900012],
-                          [-116.46422312599977, 46.28768217800006]], bounds
+        assert bounds == [[41.99187135900012, -124.56617536999985,],
+                          [46.28768217800006, -116.46422312599977]], bounds
```
Comment (BibMartin) [inline tests/test_folium.py]: To pass the tests, you just need to remove extra comma after `-124.56617536999985`.

**Comment by BibMartin** [comment]
Thanks @eddies . Clean and awesome PR.
I'll merge it as soon as you've passed the tests (see comment in diff).

@ocefpaf This worth being cherry-picked on `v0.2` branch, no ? Can you do it when it's merged on master, please ?

**Comment by eddies** [comment]
@BibMartin thanks for catching that--I had only ensured that `py.test -s tests/` was passing for me locally so hadn't noticed.

Seems unlikely that this latest CI failure (https://travis-ci.org/python-visualization/folium/jobs/115804229) is related to this PR. Can someone trigger a re-run?

**Comment by ocefpaf** [comment]
Great work!  Thanks for the PR! :+1: 

> Seems unlikely(?) that this latest CI failure (https://travis-ci.org/python-visualization/folium/jobs/115804229) is related to this PR. Can someone trigger a re-run?

You are correct. The PR is fine and we need to fix that in another one.

@BibMartin feel free to merge. I will prep a `v0.2.1` release with this fix.

**Comment by BibMartin** [comment]
I relaunched the tests and they pass now -> Merging. Thanks again @eddies .

**Pull request merged**